imagetest/junit.go

```diff
@@ -0,0 +1,98 @@
+package imagetest
```

Comment:
i have seen package-level comments before, but not file-level. is there a file with an example of this you're thinking of?

Reply:
There's no specific format. Since you prefer to keep multiple classes in a single file, having a top level comment describing what they do together would be helpful